pkg/cmd/roachtest/sqlalchemy.go

```diff
@@ -350,8 +345,36 @@ func registerSQLAlchemy(r *registry) {
 	}
 
 	r.Add(testSpec{
-		Name:    "sqlalchemy",
-		Cluster: makeClusterSpec(5),
+		Name:    "sqlalchemy1",
```

Comment:
This could be reduced a bit:

```
for i := 1; i <= 5; i++ {
	r.Add(testSpec{
￼			Name:    fmt.Sprintf("sqlalchemy%d", 2),
￼			Cluster: makeClusterSpec(1),
￼			Run: func(ctx context.Context, t *test, c *cluster) {
￼				runSQLAlchemy(ctx, t, c)
￼			},
￼		})
}
```